README_js.md

```diff
@@ -500,6 +500,7 @@ Prior to `uuid@11`, it was possible for `options` state to interfere with the in
 
 ### React Native / Expo
 
+<a name="getrandomvalues-not-supported"></a>
```

Comment:
Good catch, but this isn't the markdown that got removed.  Let's this and restore these lines above this section (where they were previously), please:

https://github.com/uuidjs/uuid/blob/bc46e198ab06311a9d82d3c9c6222062dd27f760/README_js.md?plain=1#L385-L387,